src/components/modal-dialog/modal-dialog.tsx

```diff
@@ -0,0 +1,118 @@
+import styled, { useTheme } from "styled-components";
+import { ActionButton } from "../action-button/action-button";
+import { ActionButtonVariant } from "../../types";
+import CloseIcon from "../../../public/icons/close.svg";
+
+const Overlay = styled.div`
+  position: fixed;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: #00000099;
```

Comment:
By figma: 
#232430 50% . Use global constants for colors.

Reply:
Done